api/schemas/organizations_mutations.py

```diff
@@ -53,9 +65,10 @@ class Arguments:
         @require_token
         def mutate(self, info, **kwargs):
             user_id = kwargs.get('user_id')
-            acronym = cleanse_input(kwargs.get('acronym'))
+            acronym = cleanse_input(kwargs.get('acronym', ''))
```

Comment:
This seems weird, but I do not know enough about it.  Why do you have you have an empty string parameter for the `kwargs.get()` function???

Reply:
This is to deal with the cleanse input functionality, if kwargs cant find the key in the dict then it returns none. The cleanse input function then has a fit about being passed a None type

Reply:
I think since `cleanse_input()` is a function that we wrote, it may be cleaner to handle this edge case in that function.  If you do not want to do it that way then I think a comment explaining the empty string would be necessary.